course/models.py

```diff
@@ -136,6 +136,13 @@ def get_student_pending(self, id: int):
     def get_student_courses(self, id: int):
         return self.filter(student_id=id)
 
+    def get_teacher_courses_that_student_is_in(self, id: User):
+        teacher_courses = StudentCourse.objects.none()
+        studentCourses = self.filter(student_id=id, status=Status.CONFIRMED)
+        for student in studentCourses.values_list():
+            teacher_courses |= TeacherCourse.objects.filter(course_id=student[2])
```

Comment:
- Why `teacher_courses` is `StudentCourse` at first and after it becomes `TeacherCourse`?
- Maybe we can use a filter instead of doing this for loop?